config.json

```diff
@@ -33,6 +33,37 @@
             "name": "Target URL",
             "required": true,
             "type": "text"
+        },
+        {
+            "key": "contact-admin-button-descriptive-text",
+            "name": "Please make sure you provide both a parameter name and a value. If one of the two is left blank, the url will not contain such parameter name/value.",
```

Comment:
How about this text "Please make sure you provide both a parameter name and a value. "

Reply:
Done.